### DISCUSSION TITLE
Multi touch support

### DISCUSSION THREAD

**Discussion by ScaredPenguinXXL** [ScratchAddons/ScratchAddons#4489] [Ideas]
Scratch currently doesn't support multi-touch thus mobile projects tend to be hard to navigate, could it be possible to have support for multiple simultaneous clicks?

**Comment by Norbiros** [comment]
You cannot install our extension on phone eaisly. Type in browser and try to install :-/ If we want to add this feature, first of all we need to spend huge amount of time making SA to work on Mobile devices, is it really worth it?

**Reply by cobaltt7** [reply]
The only mobile browsers that support extensions are Kiwi and Safari iirc. We dont support Safari. Kiwi works but why make an addon that only works for a small percentage of users? Turbowarp and other Scratch mods could use this addon, true, but they could just add it to their own fork.

**Comment by BarelySmooth** [comment]
If multitouch was supported, then it would allow the position of the cursor to be in two different positions at a time. This would cause projects using the “mouse x” and “mouse y” blocks in the sensing category to break.

**Reply by Samq64** [reply]
I don't think it needs to support multiple mouse coordinates, just things like when sprite clicked and touching mouse pointer.
